SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1851,6 +1851,11 @@ int32 USpatialNetDriver::ServerReplicateActors(float DeltaSeconds)
 	}
 	check(SpatialConnection->bReliableSpatialConnection);
 
+	if (DebugCtx != nullptr)
+	{
+		DebugCtx->TickServer();
+	}
```

Comment:
@ImprobableNic or @improbable-dmitrii what's your opinion on moving it here?
The repGraph returns on the next line so the debug interface was not running. I think it's safe to move it up here, but you will have more context.

Reply:
Looks good